### ISSUE TITLE
Update the Zero Install feed for Minikube

### ISSUE THREAD

**Issue by afbjorklund** [kubernetes/minikube#7978]
I would really like to support this way of distributing links to the minikube binaries:

https://apps.0install.net/kubernetes/minikube.xml

But currently I'm getting checksum errors, when trying to run the latest versions:

`0install run https://apps.0install.net/kubernetes/minikube.xml`

Incorrect manifest -- archive is corrupted.
Required digest: sha256new_COUKLDJ3K547BUITQTMI3KIXICLFELV5OAPPAURHJMIRIP6V2SWA
Actual digest: sha256new_IN5EGYYHA4Y7MG2JHVVIHL6ZUHVXMQSZHQ54SQHY2KPZ2N5LGNDQ

----

We should work with the Zero Install project, to make sure that it works out-of-the-box...
It has a lot of nice features, like a one-line start and automatic checksums and signatures.

See http://0install.net/ for details.

Eventually we could compress our binaries, to make for a smaller download as well...
They would shrink from the current 53M to a much leaner 23M - for the _same_ content!

23M	minikube-linux-amd64.tar.gz

**Comment by afbjorklund** [comment]
https://github.com/0install/apps/commit/080cd4c87ad72f33eaf754975b186e2d5c4eacd6

**Comment by medyagh** [comment]
can u provide docs how to use it ?

**Comment by afbjorklund** [comment]
> can u provide docs how to use it ?

They were actually included above:

`0install run https://apps.0install.net/kubernetes/minikube.xml`

A better example (that actually works):

`0install run https://apps.0install.net/kubernetes/kubectl.xml`

----

For the first time using the system, you _would_ need to install something.
But that is usually provided by the distribution, so should not be an issue...

```
Command '0install' not found, but can be installed with:

sudo apt install 0install-core

```

It will install the CLI version by default, so if you want the GUI you can add it too.
The separate package (for core) is to cut down on the number of dependencies.

`sudo apt install 0install`

![0install-gui](https://user-images.githubusercontent.com/10364051/80908835-8d2b0500-8d23-11ea-9cf9-aa01c4a68219.png)


Note that it will automatically pick the operating system and cpu architecture for you.
As well as keep it updated (you also have the option to pick a version as "preferred")

For our [regular instructions](https://minikube.sigs.k8s.io/docs/start/), the user would have to remember do all that themselves:

```
 curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
 sudo install minikube-linux-amd64 /usr/local/bin/minikube
```

And that is **not** verifying any checksums or signatures (even if that is also the problem here...)
My intution says that it is the modification time that is the issue here, so should be easy to fix.

You can see how it all works here: https://docs.0install.net/details/

Notice that it is just using our _regular_ minikube binaries in this case.

----

In fact, we provide the _exact_ same binary in multiple formats - to make it more accessible...
Whether the user downloads the deb or the rpm, or the raw elf or the tgz - it's the **same** exe:

```
make out/minikube-linux-amd64 out/minikube-linux-amd64.tar.gz \
     out/minikube-$(make rpm_version).rpm out/minikube_$(make deb_version).deb
```

Once the checksums have been fixed (this issue), I will add it to the minikube documentation.
Since we seem to be **unable** to host our own signed package repos, this could be an alternative.

* #3110 apt/deb
* #4716 yum/rpm

It is blocked by https://github.com/kubernetes/release/issues/839

Our limited workaround is #7971 (fixes the URL version part, but no signatures and no updates)

```
curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
sudo dpkg -i minikube_latest_amd64.deb
```

**Comment by afbjorklund** [comment]
>  My intution says that it is the modification time that is the issue here, so should be easy to fix.

I was close, it was the executable bit (the modification time was always set 0 for loose files)

It works OK with newer `0install`. If we add tarballs and icons, then it would work fine in all...

**Comment by afbjorklund** [comment]
This issue has been fixed since v1.10.0
